README.md

```diff
@@ -132,16 +132,32 @@ You will need credentials of the following form, named "env.list", at the top-le
 The credentials below are valid, so you can copy and use them to access a development instance of Augur.
 
 ```
-    connection_string=sqlite:///:memory:
-    database=astros
-    host=chaoss.tv
-    password=!xpk98T6?bK
-    port=5432
-    schema=augur_data
-    user=eightknot
-    user_type=read_only
+    AUGUR_DATABASE=astros
+    AUGUR_HOST=chaoss.tv
+    AUGUR_PASSWORD=!xpk98T6?bK
+    AUGUR_PORT=5432
+    AUGUR_SCHEMA=augur_data
+    AUGUR_USERNAME=eightknot
+    8KNOT_DEBUG=True
 ```
 
+If you have a companion Augur front end application you'll need to set the following credentials in the env.list as well.
+By setting these credentials, a button on the top tab of the application will become available to allow you to create an account on
+your Augur front end, to log into your application via this front end, and to create user-defined groups of repos/organizations that
+will become available in your application, prefixed by your Augur username (e.g. jkunstle_example_group and jkunstle_other_example).
```

Comment:
I would specify that jkunstle is the username

Reply:
@cdolfi done